console/src/App/App.less

```diff
@@ -53,6 +53,24 @@
 		}
 	}
 }
+:global(.carbon) {
+	.app {
+		.beforeTabs {
+			margin: 0 0.5em;
+
+			.avatar {
+				margin: 0 1em;
+			}
+		}
+		.afterTabs {
+			margin: 0 1em;
```

Comment:
nit: why not have top-margin for the clock to be placed just a bit lower?

Reply:
My aim was to remove as much space as possible and the top margin of Clock was a casualty of this. We can add some back, but the more we add, the more likely it will be to cover the trip info since the Carbon header is so big (relatively speaking). I don't mind adding some back though if it's just a little bit to make it look more balanced.

Reply:
I think we have some space between the clock and the buttons. `margin-top: 0.25em` doesn't seem to hurt?

Reply:
Go for it dood!